scanpipe/models.py

```diff
@@ -1592,10 +1605,14 @@ def children(self, codebase=None):
         https://github.com/nexB/commoncode/blob/main/src/commoncode/resource.py
 
         `codebase` is not used in this context but required for compatibility
-        with the commoncode.resource.VirtualCodebase class API.
+        with the commoncode.resource.Codebase class API.
         """
-        exactly_one_sub_directory = "[^/]+$"
-        children_regex = rf"^{self.path}/{exactly_one_sub_directory}"
+        if self.path == ".":
```

Comment:
Same as above, it's not clear nor documented why we have to treat the root differently.